server/routes/validators/induction/additionalTrainingFormValidator.ts

```diff
@@ -58,10 +60,14 @@ const validateAdditionalTrainingOther = (
 
   const { additionalTraining, additionalTrainingOther } = additionalTrainingForm
 
-  if (additionalTraining && additionalTraining.includes(AdditionalTrainingValue.OTHER) && !additionalTrainingOther) {
-    errors.push(
-      `Enter the type of training or vocational qualification ${prisonerSummary.firstName} ${prisonerSummary.lastName} has`,
-    )
+  if (additionalTraining && additionalTraining.includes(AdditionalTrainingValue.OTHER)) {
```

Comment:
This is where the bulk of the functional change is. We are updating the existing Induction form validators so that they not only validate for the presence of fields, but now also validate the length of the "other" text fields